pkg/psmdb/mongo/mongo.go

```diff
@@ -81,6 +81,85 @@ func ReadConfig(ctx context.Context, client *mongo.Client) (RSConfig, error) {
 	return *resp.Config, nil
 }
 
+func CreateRole(ctx context.Context, client *mongo.Client, role string, privileges []interface{}, roles []interface{}) error {
+	resp := OKResponse{}
+
+	privilegesArr := bson.A{}
+	for _, p := range privileges {
+		privilegesArr = append(privilegesArr, p)
+	}
+
+	rolesArr := bson.A{}
+	for _, r := range roles {
+		rolesArr = append(rolesArr, r)
+	}
+
+	m := bson.D{
+		{Key: "createRole", Value: role},
+		{Key: "privileges", Value: privilegesArr},
+		{Key: "roles", Value: rolesArr},
+	}
+
+	res := client.Database("admin").RunCommand(context.Background(), m)
+	if res.Err() != nil {
+		return errors.Wrap(res.Err(), "failed to create role")
+	}
+
+	err := res.Decode(&resp)
+	if err != nil {
+		return errors.Wrap(err, "failed to decode response")
+	}
+
+	if resp.OK != 1 {
+		return errors.Errorf("mongo says: %s", resp.Errmsg)
+	}
+
+	return nil
+}
+
+func CreateUser(ctx context.Context, client *mongo.Client, user, pwd string, roles []interface{}) error {
```

Comment:
you can change `roles []interface{}` just to `roles ...interface{}` to get rid of []interface{}{"clusterMonitor"}

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/548/commits/681651964d009c7ec4b0b95c1a4a7d902f4633ef